functions/accept.js

```diff
@@ -69,10 +73,20 @@ module.exports.post = async function(event, context) {
     Body: image.data
   }).promise();
 
+  const upstreamServiceUrl =
+    UPSTREAM_SERVICE_URL !== "__MOCK__"
```

Comment:
Kind of a hack here, makes the queue processing use a loop-back mock service URL on a hardcoded value in development